<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BISHT" ignored="0" name="Validate Title" tests="2" failures="1" timestamp="25 Apr 2019 16:51:12 GMT" time="10.519" errors="0">
  <testcase name="doLogin" time="10.491" classname="TestNG.TestCase2">
    <failure type="java.lang.AssertionError" message="user not logged in successfully">
      <![CDATA[java.lang.AssertionError: user not logged in successfully
at TestNG.TestCase2.doLogin(TestCase2.java:19)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="ValidateTitle" time="0.001" classname="TestNG.TestCase2">
    <skipped/>
  </testcase> <!-- ValidateTitle -->
</testsuite> <!-- Validate Title -->
